site/gdocs/components/ArticleBlock.tsx

```diff
@@ -411,6 +411,14 @@ function ArticleBlockInternal({
                 dangerouslySetInnerHTML={{ __html: block.value }}
             />
         ))
+        .with({ type: "script" }, (block) => (
+            <div
+                className={getLayout("script", containerType)}
+                dangerouslySetInnerHTML={{
+                    __html: `<script type="module">${block.value.value}</script>`,
```

Comment:
I don't know why I have to use `.value.value` here.

I've checked, and the block really gets read into `{value: {value: "console.log('test');" } }`. But I don't know why that is.

@ikesau do you know how to fix that?